Makefile

```diff
@@ -12,8 +12,7 @@ venv:
 
 setup:
 	@pip install -U -e .[dev]
-	@npm install
-	@pre-commit install
+  @npm install
```

Comment:
Formatting?

Reply:
Make file won't run until you fix the indentation - test it.

Reply:
Done